articles/130_ros_time.md

```diff
@@ -140,6 +144,15 @@ The underlying datatypes will also provide ways to register notifications, howev
   <b>TODO:</b> Enumerate the <code>rcl</code> datastructures and methods here.
 </div>
 
+## Work Needing To Be Done (Apr 2017)
+
+- Decide where and when to autosubscribe to the `/clock` message. Built in to Node? A Node addon or service? Part of the first call to `rcl_node_init`?
```

Comment:
You're probably thinking of my trailing comments here:
https://discourse.ros.org/t/of-clocks-and-simulation-betimes-and-otherwise/1587

The decision boils down to this: build it in to ROS node or opt-in to ROS Time by programmatically constructing an additional object? The same question goes for the ROS 1 node and system parameters, or maybe we say use command line parameters for all those usages. Maybe we want to use the command line parameters passed to rclcpp::init to change the default clock subscription mechanism. I'm okay with any of those options. The critical point is this: if we can make a decision on this very soon, I will have some time to implement it as part of my work for a June software release. I'd be happy to set up a conference call for discussing these options.